app/assets/stylesheets/modules/sitter.scss

```diff
@@ -2,20 +2,31 @@
 @import "mixins";
 
 //SEARCH
+.all {
+  max-width: 80%;
+
+  #map-canvas {
+    height: 1000px;
```

Comment:
Properties should be ordered align, float, height, left, margin-right, margin-top, max-width, position, width, z-index